drivers/pwm/pwm_stm32.c

```diff
@@ -270,9 +278,17 @@ static int pwm_stm32_pin_set(const struct device *dev, uint32_t pwm,
 		LL_TIM_OC_StructInit(&oc_init);
 
 		oc_init.OCMode = LL_TIM_OCMODE_PWM1;
-		oc_init.OCState = LL_TIM_OCSTATE_ENABLE;
+
```

Comment:
Few line above channel is defined like that 
`channel = ch2ll[pwm - 1u];`
it is then used in LL API like` LL_TIM_CC_DisableChannel()`, `LL_TIM_CC_IsEnabledChannel().`
But those API accept Complementary channel. So, is case of complementary, `ch2ll_n[]` should be used